src/vmm/src/builder.rs

```diff
@@ -552,6 +573,44 @@ pub fn setup_serial_device(
     Ok(serial)
 }
 
+/// 64 bytes due to alignment requirement in 3.1 of https://www.kernel.org/doc/html/v5.8/virt/kvm/devices/vcpu.html#attribute-kvm-arm-vcpu-pvtime-ipa
+#[cfg(target_arch = "aarch64")]
+const STEALTIME_STRUCT_MEM_SIZE: u64 = 64;
+
+/// Helper method to allocate steal time region
+#[cfg(target_arch = "aarch64")]
+fn allocate_pvtime_region(
+    vmm: &mut Vmm,
+    vcpu_count: usize,
+    policy: vm_allocator::AllocPolicy,
```

Comment:
If you drop the call to this on the restore path, you can also drop this `policy` argument (and just inline this into `setup_pvtime` probably).